client/src/app/modules/welcome/welcome.component.scss

```diff
@@ -0,0 +1,79 @@
+$orange-color: #F5612C;
+$blue-color: #2DADB7;
+.welcome-body {
+    height: 100vh;
+    width: 100%;
+    background-image: url(../../../assets/images/background/welcome-background.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.website-title {
+    font-family: cursive;
```

Comment:
font-family should apply in style.scss file because the whole website should follow one specific font not an individual section